myia/ir/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Main exports for ir submodule."""
+
+from .abstract import Node  # noqa
+from .anf import Graph, ANFNode, Apply, Constant, Parameter, Special  # noqa
+from .clone import GraphCloner  # noqa
+from .utils import (  # noqa
+    succ_deep, succ_deeper, succ_incoming, succ_bidirectional,
+    exclude_from_set, freevars_boundary,
+    dfs, toposort, accessible_graphs,
+    destroy_disconnected_nodes,
+    isomorphic,
+    replace,
+    is_apply, is_parameter, is_constant, is_constant_graph, is_special
+)
```

Comment:
Rather than `# noqa`-ing all the things, it's cleaner to define `__all__`.

Reply:
I suppose, although I find it annoyingly redundant.

Reply:
I don't like `__all__` since we have to define everything twice.  The solution in Theano was to whitelist all the `__init__.py` files, but `#noqa` for the imports is better I think.

It would be nice if we could just silence that warning, but it's the same as the one for regular files.